cmd/web.go

```diff
@@ -495,6 +495,7 @@ func runWeb(ctx *cli.Context) {
 					Post(bindIgnErr(auth.NewWikiForm{}), repo.NewWikiPost)
 				m.Combo("/:page/_edit").Get(repo.EditWiki).
 					Post(bindIgnErr(auth.NewWikiForm{}), repo.EditWikiPost)
+				m.Post("/:page", bindIgnErr(auth.NewWikiForm{}), repo.DeleteWikiPagePost)
```

Comment:
Actually, based on the js modal it only supports post and I don't want to get into refactoring all of that right now. I'll update the route to POST to /delete :)